src/main/java/com/aws/greengrass/shadowmanager/ipc/UpdateThingShadowRequestHandler.java

```diff
@@ -235,6 +242,8 @@ public UpdateThingShadowHandlerResponse handleRequest(UpdateThingShadowRequest r
                     this.syncHandler.pushCloudUpdateSyncRequest(thingName, shadowName, updateDocumentRequest);
 
                     return new UpdateThingShadowHandlerResponse(updateThingShadowResponse, updateDocumentBytes);
+                } catch (InvalidRequestParametersException e) {
```

Comment:
we want throw back the correct exception message here right?  `ErrorMessage.PAYLOAD_TOO_LARGE_MESSAGE`